weave/trace_server/trace_server_interface.py

```diff
@@ -19,11 +19,25 @@ class ExtraKeysTypedDict(TypedDict):
 # https://docs.pydantic.dev/2.8/concepts/strict_mode/#dataclasses-and-typeddict
 ExtraKeysTypedDict.__pydantic_config__ = ConfigDict(extra="allow")  # type: ignore
 
-
-class WeaveSummarySchema(ExtraKeysTypedDict):
-    status: typing.Optional[typing.Literal["success", "error", "running"]]
-    nice_trace_name: typing.Optional[str]
-    latency: typing.Optional[int]
+cost_string_fields = [
```

Comment:
I would recommend moving these lists into your DB layer. I think they are more descriptive of the on-disk state than the over-the-wire type